### PR TITLE
Update/packages/atom  button  #wa 341

### PR DESCRIPTION
**Pull request by Mar0cas**
- Update component Config files
- Update component SRC file, importing stylesheet
- Update component Mock files
- Update component Stories files
- Update component Test files
- Component Style SRC files
- Component Style Mixin files
- Component Style Variable files

### CODE REVIEW

File: packages/atoms/button/src/styles/_link.scss [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    font-size: pxrem(13);
+    font-weight: $semi-bold;
+    text-transform: uppercase;
+    display: inline-flex;
+    align-items: center;
+    justify-content: flex-start;
+    // color: var(--color-darblue);
+    transition: 100ms color ease-out;
+
+    // &:hover,
```
Comment (LuisConceicaoTD) [inline packages/atoms/button/src/styles/_link.scss:19]: Are these comments needed for later?

File: packages/atoms/button/src/styles/mixins/_button.scss [hunk 2]
```diff
@@ -0,0 +1,108 @@
+// @mixin make-button-outlined-variation($variations, $name) {
```
Comment (LuisConceicaoTD) [inline packages/atoms/button/src/styles/mixins/_button.scss:1]: Same question for this commented block

**Review by LuisConceicaoTD — APPROVED** [review]

File: packages/atoms/button/src/styles/_link.scss (hunk 1 above)
Comment (Mar0cas) [reply packages/atoms/button/src/styles/_link.scss:19]: For now, I just commented them, but probably in the new style rework, we will use them.

Comment (Mar0cas) [reply packages/atoms/button/src/styles/_link.scss:19]: For now, I just commented them, but probably in the new style rework, we will use them.

File: packages/atoms/button/src/styles/mixins/_button.scss (hunk 2 above)
Comment (Mar0cas) [reply packages/atoms/button/src/styles/mixins/_button.scss:1]: We still don't know if we will have a dark and light variant to the outlined buttons. So I will leave them there for future use.

**Pull request merged**